.github/ISSUE_TEMPLATE/03_question.md

```diff
@@ -0,0 +1,10 @@
+---
+name: "❔ Request"
+about: A question about the tooling, a template, ...
```

Comment:
Question should probably be redirected to [Github discussions](https://docs.github.com/en/discussions)